src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -1,118 +1,29 @@
 package traffic_simulation
 
 class Vehicle(val id: Int, val wannaDriveInHours: MutableList<Int>) {
-    val delayedInHours: MutableList<Int> = mutableListOf() //no vehicle is delayedInHours by default
```

Comment:
Es gab schon in Block 2 den Auftrag, auszugeben ob Verspätung war. 

Und es hieß mal [hier](https://github.com/RF-Buerky/BBFW-Project/pull/41/files/59312840db62967e5e482614517d8b4e363e8e85#diff-1b5c030d3c79b51357f662e7041de7ed)
> I didn't mention replacing it and future tasks actually extend it.

😅